### ISSUE TITLE
OpenAPI definition includes strange SystemFile line

### ISSUE THREAD

**Issue by sdelamo** [micronaut-projects/micronaut-aws#1843]
### Issue description

Sample app: 

[projectcheckins.zip](https://github.com/micronaut-projects/micronaut-aws/files/12394012/projectcheckins.zip)

Steps to reproduce: 

Download sample app: 

```
 ./gradlew test 

> Task :app:compileJava
Note: Generating OpenAPI Documentation
Note: Writing OpenAPI file to destination: /Users/sdelamo/github/softamo/projectcheckins/app/build/classes/java/main/META-INF/swagger/application.yml
Note: Writing OpenAPI views to destination: /Users/sdelamo/github/softamo/projectcheckins/app/build/classes/java/main/META-INF/swagger/views
Note: Creating bean classes for 2 type elements

> Task :app:compileTestJava
Note: Creating bean classes for 1 type elements

> Task :app:test FAILED

HomeControllerTest > testOpenAPI() FAILED
    org.opentest4j.AssertionFailedError at HomeControllerTest.java:49
```

**Comment by timyates** [comment]
I believe this should be fixed by https://github.com/micronaut-projects/micronaut-servlet/pull/550

**Comment by sdelamo** [comment]
I have released servlet 4.0.5. @timyates Can you create a PR which adds a test and updates servlet to 4.0.5

**Comment by timyates** [comment]
@sdelamo https://github.com/micronaut-projects/micronaut-aws/pull/1845 👍
